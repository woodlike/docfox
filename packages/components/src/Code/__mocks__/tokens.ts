import { Token } from 'prismjs';

export const prismTokens = [
  {
    type: 'tag',
    content: [
      {
        type: 'tag',
        content: [
          {
            type: 'punctuation',
            content: '<',
            length: 1,
            greedy: false,
          },
          'pre',
        ],
        length: 4,
        greedy: false,
      },
      ' ',
      {
        type: 'attr-name',
        content: ['sx'],
        length: 2,
        greedy: false,
      },
      {
        type: 'script',
        content: [
          {
            type: 'script-punctuation',
            content: '=',
            alias: 'punctuation',
            length: 1,
            greedy: false,
          },
          {
            type: 'punctuation',
            content: '{',
            length: 1,
            greedy: false,
          },
          {
            type: 'function',
            content: 'createStylesPre',
            length: 15,
            greedy: false,
          },
          {
            type: 'punctuation',
            content: '(',
            length: 1,
            greedy: false,
          },
          'props',
          {
            type: 'punctuation',
            content: '.',
            length: 1,
            greedy: false,
          },
          'size',
          {
            type: 'punctuation',
            content: ',',
            length: 1,
            greedy: false,
          },
          ' qt',
          {
            type: 'punctuation',
            content: ',',
            length: 1,
            greedy: false,
          },
          ' props',
          {
            type: 'punctuation',
            content: '.',
            length: 1,
            greedy: false,
          },
          'theme',
          {
            type: 'punctuation',
            content: ')',
            length: 1,
            greedy: false,
          },
          {
            type: 'punctuation',
            content: '}',
            length: 1,
            greedy: false,
          },
        ],
        alias: 'language-javascript',
        length: 47,
        greedy: false,
      },
      {
        type: 'punctuation',
        content: '>',
        length: 1,
        greedy: false,
      },
    ],
    length: 55,
    greedy: true,
  },
  '\n    ',
  {
    type: 'tag',
    content: [
      {
        type: 'tag',
        content: [
          {
            type: 'punctuation',
            content: '<',
            length: 1,
            greedy: false,
          },
          'code',
        ],
        length: 5,
        greedy: false,
      },
      ' ',
      {
        type: 'attr-name',
        content: ['sx'],
        length: 2,
        greedy: false,
      },
      {
        type: 'script',
        content: [
          {
            type: 'script-punctuation',
            content: '=',
            alias: 'punctuation',
            length: 1,
            greedy: false,
          },
          {
            type: 'punctuation',
            content: '{',
            length: 1,
            greedy: false,
          },
          'stylesCode',
          {
            type: 'punctuation',
            content: '}',
            length: 1,
            greedy: false,
          },
        ],
        alias: 'language-javascript',
        length: 13,
        greedy: false,
      },
      {
        type: 'punctuation',
        content: '>',
        length: 1,
        greedy: false,
      },
    ],
    length: 22,
    greedy: true,
  },
  '\n      ',
  {
    type: 'punctuation',
    content: '{',
    length: 1,
    greedy: false,
  },
  'tokens',
  {
    type: 'punctuation',
    content: '.',
    length: 1,
    greedy: false,
  },
  {
    type: 'function',
    content: 'map',
    length: 3,
    greedy: false,
  },
  {
    type: 'punctuation',
    content: '(',
    length: 1,
    greedy: false,
  },
  {
    type: 'punctuation',
    content: '(',
    length: 1,
    greedy: false,
  },
  'token',
  {
    type: 'operator',
    content: ':',
    length: 1,
    greedy: false,
  },
  ' Token ',
  {
    type: 'operator',
    content: '|',
    length: 1,
    greedy: false,
  },
  ' string',
  {
    type: 'punctuation',
    content: ',',
    length: 1,
    greedy: false,
  },
  ' i',
  {
    type: 'operator',
    content: ':',
    length: 1,
    greedy: false,
  },
  ' number',
  {
    type: 'punctuation',
    content: ')',
    length: 1,
    greedy: false,
  },
  {
    type: 'operator',
    content: ':',
    length: 1,
    greedy: false,
  },
  ' ',
  {
    type: 'constant',
    content: 'JSX',
    length: 3,
    greedy: false,
  },
  {
    type: 'punctuation',
    content: '.',
    length: 1,
    greedy: false,
  },
  {
    type: 'parameter',
    content: ['Element'],
    length: 7,
    greedy: false,
  },
  ' ',
  {
    type: 'operator',
    content: '=>',
    length: 2,
    greedy: false,
  },
  ' ',
  {
    type: 'punctuation',
    content: '{',
    length: 1,
    greedy: false,
  },
  '\n        console',
  {
    type: 'punctuation',
    content: '.',
    length: 1,
    greedy: false,
  },
  {
    type: 'function',
    content: 'log',
    length: 3,
    greedy: false,
  },
  {
    type: 'punctuation',
    content: '(',
    length: 1,
    greedy: false,
  },
  'tokens',
  {
    type: 'punctuation',
    content: ')',
    length: 1,
    greedy: false,
  },
  {
    type: 'punctuation',
    content: ';',
    length: 1,
    greedy: false,
  },
  '\n        ',
  {
    type: 'keyword',
    content: 'return',
    length: 6,
    greedy: false,
  },
  ' token',
  {
    type: 'punctuation',
    content: '.',
    length: 1,
    greedy: false,
  },
  'type ',
  {
    type: 'operator',
    content: '?',
    length: 1,
    greedy: false,
  },
  ' ',
  {
    type: 'punctuation',
    content: '(',
    length: 1,
    greedy: false,
  },
  '\n          Array',
  {
    type: 'punctuation',
    content: '.',
    length: 1,
    greedy: false,
  },
  {
    type: 'function',
    content: 'isArray',
    length: 7,
    greedy: false,
  },
  {
    type: 'punctuation',
    content: '(',
    length: 1,
    greedy: false,
  },
  'token',
  {
    type: 'punctuation',
    content: '.',
    length: 1,
    greedy: false,
  },
  'content',
  {
    type: 'punctuation',
    content: ')',
    length: 1,
    greedy: false,
  },
  ' ',
  {
    type: 'operator',
    content: '?',
    length: 1,
    greedy: false,
  },
  ' ',
  {
    type: 'punctuation',
    content: '(',
    length: 1,
    greedy: false,
  },
  '\n            ',
  {
    type: 'tag',
    content: [
      {
        type: 'tag',
        content: [
          {
            type: 'punctuation',
            content: '<',
            length: 1,
            greedy: false,
          },
          {
            type: 'class-name',
            content: 'TokenList',
            length: 9,
            greedy: false,
          },
        ],
        length: 10,
        greedy: false,
      },
      ' ',
      {
        type: 'attr-name',
        content: ['content'],
        length: 7,
        greedy: false,
      },
      {
        type: 'script',
        content: [
          {
            type: 'script-punctuation',
            content: '=',
            alias: 'punctuation',
            length: 1,
            greedy: false,
          },
          {
            type: 'punctuation',
            content: '{',
            length: 1,
            greedy: false,
          },
          'token',
          {
            type: 'punctuation',
            content: '.',
            length: 1,
            greedy: false,
          },
          'content',
          {
            type: 'punctuation',
            content: '}',
            length: 1,
            greedy: false,
          },
        ],
        alias: 'language-javascript',
        length: 16,
        greedy: false,
      },
      ' ',
      {
        type: 'attr-name',
        content: ['theme'],
        length: 5,
        greedy: false,
      },
      {
        type: 'script',
        content: [
          {
            type: 'script-punctuation',
            content: '=',
            alias: 'punctuation',
            length: 1,
            greedy: false,
          },
          {
            type: 'punctuation',
            content: '{',
            length: 1,
            greedy: false,
          },
          'theme',
          {
            type: 'punctuation',
            content: '}',
            length: 1,
            greedy: false,
          },
        ],
        alias: 'language-javascript',
        length: 8,
        greedy: false,
      },
      ' ',
      {
        type: 'punctuation',
        content: '/>',
        length: 2,
        greedy: false,
      },
    ],
    length: 51,
    greedy: true,
  },
  '\n          ',
  {
    type: 'punctuation',
    content: ')',
    length: 1,
    greedy: false,
  },
  ' ',
  {
    type: 'operator',
    content: ':',
    length: 1,
    greedy: false,
  },
  ' ',
  {
    type: 'punctuation',
    content: '(',
    length: 1,
    greedy: false,
  },
  '\n            ',
  {
    type: 'tag',
    content: [
      {
        type: 'tag',
        content: [
          {
            type: 'punctuation',
            content: '<',
            length: 1,
            greedy: false,
          },
          'span',
        ],
        length: 5,
        greedy: false,
      },
      ' ',
      {
        type: 'attr-name',
        content: ['sx'],
        length: 2,
        greedy: false,
      },
      {
        type: 'script',
        content: [
          {
            type: 'script-punctuation',
            content: '=',
            alias: 'punctuation',
            length: 1,
            greedy: false,
          },
          {
            type: 'punctuation',
            content: '{',
            length: 1,
            greedy: false,
          },
          'theme',
          {
            type: 'punctuation',
            content: '.',
            length: 1,
            greedy: false,
          },
          {
            type: 'function',
            content: 'get',
            length: 3,
            greedy: false,
          },
          {
            type: 'punctuation',
            content: '(',
            length: 1,
            greedy: false,
          },
          'token',
          {
            type: 'punctuation',
            content: '.',
            length: 1,
            greedy: false,
          },
          'type',
          {
            type: 'punctuation',
            content: ')',
            length: 1,
            greedy: false,
          },
          {
            type: 'punctuation',
            content: '}',
            length: 1,
            greedy: false,
          },
        ],
        alias: 'language-javascript',
        length: 24,
        greedy: false,
      },
      {
        type: 'punctuation',
        content: '>',
        length: 1,
        greedy: false,
      },
    ],
    length: 33,
    greedy: true,
  },
  '\n              ',
  {
    type: 'punctuation',
    content: '{',
    length: 1,
    greedy: false,
  },
  {
    type: 'keyword',
    content: 'typeof',
    length: 6,
    greedy: false,
  },
  ' token',
  {
    type: 'punctuation',
    content: '.',
    length: 1,
    greedy: false,
  },
  'content ',
  {
    type: 'operator',
    content: '===',
    length: 3,
    greedy: false,
  },
  ' ',
  {
    type: 'string',
    content: "'string'",
    length: 8,
    greedy: true,
  },
  ' ',
  {
    type: 'operator',
    content: '&&',
    length: 2,
    greedy: false,
  },
  ' token',
  {
    type: 'punctuation',
    content: '.',
    length: 1,
    greedy: false,
  },
  'content',
  {
    type: 'punctuation',
    content: '}',
    length: 1,
    greedy: false,
  },
  '\n            ',
  {
    type: 'tag',
    content: [
      {
        type: 'tag',
        content: [
          {
            type: 'punctuation',
            content: '</',
            length: 2,
            greedy: false,
          },
          'span',
        ],
        length: 6,
        greedy: false,
      },
      {
        type: 'punctuation',
        content: '>',
        length: 1,
        greedy: false,
      },
    ],
    length: 7,
    greedy: true,
  },
  '\n          ',
  {
    type: 'punctuation',
    content: ')',
    length: 1,
    greedy: false,
  },
  '\n        ',
  {
    type: 'punctuation',
    content: ')',
    length: 1,
    greedy: false,
  },
  ' ',
  {
    type: 'operator',
    content: ':',
    length: 1,
    greedy: false,
  },
  ' ',
  {
    type: 'punctuation',
    content: '(',
    length: 1,
    greedy: false,
  },
  '\n          ',
  {
    type: 'tag',
    content: [
      {
        type: 'tag',
        content: [
          {
            type: 'punctuation',
            content: '<',
            length: 1,
            greedy: false,
          },
          'span',
        ],
        length: 5,
        greedy: false,
      },
      {
        type: 'punctuation',
        content: '>',
        length: 1,
        greedy: false,
      },
    ],
    length: 6,
    greedy: true,
  },
  {
    type: 'punctuation',
    content: '{',
    length: 1,
    greedy: false,
  },
  'token',
  {
    type: 'punctuation',
    content: '}',
    length: 1,
    greedy: false,
  },
  {
    type: 'tag',
    content: [
      {
        type: 'tag',
        content: [
          {
            type: 'punctuation',
            content: '</',
            length: 2,
            greedy: false,
          },
          'span',
        ],
        length: 6,
        greedy: false,
      },
      {
        type: 'punctuation',
        content: '>',
        length: 1,
        greedy: false,
      },
    ],
    length: 7,
    greedy: true,
  },
  '\n        ',
  {
    type: 'punctuation',
    content: ')',
    length: 1,
    greedy: false,
  },
  {
    type: 'punctuation',
    content: ';',
    length: 1,
    greedy: false,
  },
  '\n      ',
  {
    type: 'punctuation',
    content: '}',
    length: 1,
    greedy: false,
  },
  {
    type: 'punctuation',
    content: ')',
    length: 1,
    greedy: false,
  },
  {
    type: 'punctuation',
    content: '}',
    length: 1,
    greedy: false,
  },
  '\n    ',
  {
    type: 'tag',
    content: [
      {
        type: 'tag',
        content: [
          {
            type: 'punctuation',
            content: '</',
            length: 2,
            greedy: false,
          },
          'code',
        ],
        length: 6,
        greedy: false,
      },
      {
        type: 'punctuation',
        content: '>',
        length: 1,
        greedy: false,
      },
    ],
    length: 7,
    greedy: true,
  },
  '\n  ',
  {
    type: 'tag',
    content: [
      {
        type: 'tag',
        content: [
          {
            type: 'punctuation',
            content: '</',
            length: 2,
            greedy: false,
          },
          'pre',
        ],
        length: 5,
        greedy: false,
      },
      {
        type: 'punctuation',
        content: '>',
        length: 1,
        greedy: false,
      },
    ],
    length: 6,
    greedy: true,
  },
] as (string | Token)[];
