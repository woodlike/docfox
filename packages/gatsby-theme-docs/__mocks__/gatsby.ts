export const gatsby = {
  ...jest.requireActual('gatsby'),
  graphql: jest.fn(),
  StaticQuery: jest.fn(),
  useStaticQuery: jest.fn(),
  reporter: {
    panicOnBuild: jest.fn().mockName('panicOnBuild'),
  },
};
