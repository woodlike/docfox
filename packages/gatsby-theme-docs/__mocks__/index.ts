export * from './data';
export * from './gatsby';
export * from './mdxAstMock';
