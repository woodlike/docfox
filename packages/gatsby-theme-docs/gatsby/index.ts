export * from './docs';
export * from './create-pages';
export * from './file';
export * from './mdx';
export * from './pre-bootstrap';
export * from './schema';
export * from './source-nodes';
export * from './utils';
