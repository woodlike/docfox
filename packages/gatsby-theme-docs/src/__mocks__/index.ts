export * from './data';
export * from './docs';
export * from './gatsby';
export * from './mdxAstMock';
