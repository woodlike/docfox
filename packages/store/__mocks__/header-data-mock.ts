export const headerQueryMock = {
  allDataJson: {
    nodes: [
      {
        cart: {
          handle: 'cart',
        },
        logo: {
          desc: 'Go back to the main page',
          title: 'Woodlike Ocean Sustainable Swimwear',
        },
        miniCart: {
          items: [
            {
              handle: 'account/login',
              title: 'Log in',
              id: '4vnwZnPaFu'
            },
            {
              handle: 'account/register',
              title: 'Register',
              id: 's4MSmQgXaG'
            },
          ],
        },
      },
    ],
  },
  site: {
    siteMetadata: {
      siteUrl: 'https://woodlikeocean.com',
    },
  }
};
