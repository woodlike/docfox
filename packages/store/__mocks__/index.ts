export * from './navigation-data-mock';
export * from './header-data-mock';
