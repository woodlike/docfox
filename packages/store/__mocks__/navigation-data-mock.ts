export const navigationQueryMock = {
  site: {
    siteMetadata: {
      siteUrl: 'https://woodlikeocean.com',
    },
  },
  allNavigationJson: {
    nodes: [
      {
        title: 'Bikinis',
        handle: 'collections/bikinis',
        id: 'w9XpzD45',
        menuItems: [
          {
            handle: 'collections/bikini-tops',
            title: 'Bikini Tops',
            id: 'gqv6qn1f'
          },
          {
            handle: 'collections/bikini-bottoms',
            title: 'Bikini Bottoms',
            id: 'SffcMKCN'
          },
          {
            handle: 'collections/bikini-sale',
            title: 'Bikini-Sale',
            id: '-aqvsu8_'
          },
          {
            handle: 'products/woodlike-ocean-gift-card-50-00-postcard',
            title: 'Gift card 50',
            id: 'lZl-O7bb'
          },
          {
            handle: 'products/woodlike-ocean-gift-card-100-00-postcard',
            title: 'Gift card 100',
            id: 'QAKJrS1d'
          },
          {
            handle: 'products/woodlike-ocean-gift-card-150-00-print-version',
            title: 'Gift card 150',
            id: '17lwT_y9'
          }
        ],
      },
      {
        title: 'One Pieces',
        handle: 'collections/one-pieces',
        id: 'VD8KXYJ7',
        menuItems: [
          {
            handle: 'collections/swimsuits-sale',
            title: 'Swimsuits Sale',
            id: 'BJ2XAIo'
          },
          {
            handle: 'products/woodlike-ocean-gift-card-50-00-postcard',
            title: 'Gift card 50',
            id: 'RIZFrAk'
          },
          {
            handle: 'products/woodlike-ocean-gift-card-100-00-postcard',
            title: 'Gift card 100',
            id: 'IMkLrAJ'
          },
          {
            handle: 'products/woodlike-ocean-gift-card-150-00-print-version',
            title: 'Gift card 150',
            id: 'Y2T8-UD'
          }
        ],
      },
      {
        title: 'Movement',
        handle: 'collections/movement',
        id: 'M_NC_Tka',
        menuItems: [
          {
            handle: 'collections/movement/tops',
            title: 'Tops',
            id: 'hlOPoHPy'
          },
          {
            handle: 'collections/movement/bottoms',
            title: 'Bottoms',
            id: 'Qrc4vg2T'
          },
          {
            handle: 'collections/movement/one-piece',
            title: 'One Pieces',
            id: 'cXI9NRXI'
          },
          {
            handle: 'products/woodlike-ocean-gift-card-50-00-postcard',
            title: 'Gift card 50',
            id: '-17sFmV7'
          },
          {
            handle: 'products/woodlike-ocean-gift-card-100-00-postcard',
            title: 'Gift card 100',
            id: 'C_il2Rz1'
          },
          {
            handle: 'products/woodlike-ocean-gift-card-150-00-print-version',
            title: 'Gift card 150',
            id: 'NZcDa-3b'
          }
        ],
      },
      {
        title: 'Shape Effect',
        handle: 'collections/shape-effect',
        id: 'UKvhrFRp',
        menuItems: [
          {
            handle: 'collections/shape-effect/tops',
            title: 'Tops',
            id: 'Fz525XDs'
          },
          {
            handle: 'collections/shape-effect/bottoms',
            title: 'Bottoms',
            id: 'j5PeZjFI'
          },
          {
            handle: 'collections/shape-effect/one-piece',
            title: 'One Pieces',
            id: 'nkUA34pk'
          },
          {
            handle: 'products/woodlike-ocean-gift-card-50-00-postcard',
            title: 'Gift card 50',
            id: 'LAFze9G-'
          },
          {
            handle: 'products/woodlike-ocean-gift-card-100-00-postcard',
            title: 'Gift card 100',
            id: 'mv4BYSmc'
          },
          {
            handle: 'products/woodlike-ocean-gift-card-150-00-print-version',
            title: 'Gift card 150',
            id: '3H2ZPBL1'
          }
        ],
      },
      {
        title: 'The Brand',
        handle: 'pages/woodlike-ocean-the-brand',
        id: 'ZVnTTk9g',
        menuItems: [
          {
            handle: 'pages/sustainability',
            title: 'Sustainability',
            id: 'HU9Ax2o0'
          },
          {
            handle: 'pages/about-us',
            title: 'About Us',
            id: 'vOGf14mm'
          }
        ],
      },
    ],
  },
};
