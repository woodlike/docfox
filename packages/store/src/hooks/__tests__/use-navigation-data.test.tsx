import { renderHook } from '@testing-library/react-hooks';
import { useStaticQuery } from 'gatsby';

import {useNavigationData} from '..';
import { navigationQueryMock } from '../../../__mocks__';

beforeEach(() => {
  (useStaticQuery as jest.Mock).mockReturnValue(navigationQueryMock);
});

describe('useNavigationData()', () => {
  it('should return an object containing the navigation data', () => {
    const { result } = renderHook(() => useNavigationData());
    expect(result.current).toMatchSnapshot();
  });
});
