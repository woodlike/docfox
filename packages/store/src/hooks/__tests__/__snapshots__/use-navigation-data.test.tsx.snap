// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useNavigationData() should return an object containing the navigation data 1`] = `
Object {
  "items": Array [
    Object {
      "handle": "collections/bikinis",
      "id": "w9XpzD45",
      "menuItems": Array [
        Object {
          "handle": "collections/bikini-tops",
          "id": "gqv6qn1f",
          "title": "Bikini Tops",
        },
        Object {
          "handle": "collections/bikini-bottoms",
          "id": "SffcMKCN",
          "title": "Bikini Bottoms",
        },
        Object {
          "handle": "collections/bikini-sale",
          "id": "-aqvsu8_",
          "title": "Bikini-Sale",
        },
        Object {
          "handle": "products/woodlike-ocean-gift-card-50-00-postcard",
          "id": "lZl-O7bb",
          "title": "Gift card 50",
        },
        Object {
          "handle": "products/woodlike-ocean-gift-card-100-00-postcard",
          "id": "QAKJrS1d",
          "title": "Gift card 100",
        },
        Object {
          "handle": "products/woodlike-ocean-gift-card-150-00-print-version",
          "id": "17lwT_y9",
          "title": "Gift card 150",
        },
      ],
      "title": "Bikinis",
    },
    Object {
      "handle": "collections/one-pieces",
      "id": "VD8KXYJ7",
      "menuItems": Array [
        Object {
          "handle": "collections/swimsuits-sale",
          "id": "BJ2XAIo",
          "title": "Swimsuits Sale",
        },
        Object {
          "handle": "products/woodlike-ocean-gift-card-50-00-postcard",
          "id": "RIZFrAk",
          "title": "Gift card 50",
        },
        Object {
          "handle": "products/woodlike-ocean-gift-card-100-00-postcard",
          "id": "IMkLrAJ",
          "title": "Gift card 100",
        },
        Object {
          "handle": "products/woodlike-ocean-gift-card-150-00-print-version",
          "id": "Y2T8-UD",
          "title": "Gift card 150",
        },
      ],
      "title": "One Pieces",
    },
    Object {
      "handle": "collections/movement",
      "id": "M_NC_Tka",
      "menuItems": Array [
        Object {
          "handle": "collections/movement/tops",
          "id": "hlOPoHPy",
          "title": "Tops",
        },
        Object {
          "handle": "collections/movement/bottoms",
          "id": "Qrc4vg2T",
          "title": "Bottoms",
        },
        Object {
          "handle": "collections/movement/one-piece",
          "id": "cXI9NRXI",
          "title": "One Pieces",
        },
        Object {
          "handle": "products/woodlike-ocean-gift-card-50-00-postcard",
          "id": "-17sFmV7",
          "title": "Gift card 50",
        },
        Object {
          "handle": "products/woodlike-ocean-gift-card-100-00-postcard",
          "id": "C_il2Rz1",
          "title": "Gift card 100",
        },
        Object {
          "handle": "products/woodlike-ocean-gift-card-150-00-print-version",
          "id": "NZcDa-3b",
          "title": "Gift card 150",
        },
      ],
      "title": "Movement",
    },
    Object {
      "handle": "collections/shape-effect",
      "id": "UKvhrFRp",
      "menuItems": Array [
        Object {
          "handle": "collections/shape-effect/tops",
          "id": "Fz525XDs",
          "title": "Tops",
        },
        Object {
          "handle": "collections/shape-effect/bottoms",
          "id": "j5PeZjFI",
          "title": "Bottoms",
        },
        Object {
          "handle": "collections/shape-effect/one-piece",
          "id": "nkUA34pk",
          "title": "One Pieces",
        },
        Object {
          "handle": "products/woodlike-ocean-gift-card-50-00-postcard",
          "id": "LAFze9G-",
          "title": "Gift card 50",
        },
        Object {
          "handle": "products/woodlike-ocean-gift-card-100-00-postcard",
          "id": "mv4BYSmc",
          "title": "Gift card 100",
        },
        Object {
          "handle": "products/woodlike-ocean-gift-card-150-00-print-version",
          "id": "3H2ZPBL1",
          "title": "Gift card 150",
        },
      ],
      "title": "Shape Effect",
    },
    Object {
      "handle": "pages/woodlike-ocean-the-brand",
      "id": "ZVnTTk9g",
      "menuItems": Array [
        Object {
          "handle": "pages/sustainability",
          "id": "HU9Ax2o0",
          "title": "Sustainability",
        },
        Object {
          "handle": "pages/about-us",
          "id": "vOGf14mm",
          "title": "About Us",
        },
      ],
      "title": "The Brand",
    },
  ],
  "url": "https://woodlikeocean.com",
}
`;
